ext/session/session.c

```diff
@@ -1681,6 +1677,11 @@ PHP_FUNCTION(session_set_cookie_params)
 		Z_PARAM_BOOL_OR_NULL(httponly, httponly_null)
 	ZEND_PARSE_PARAMETERS_END();
 
+	if (!PS(use_cookies)) {
+		php_error_docref(NULL, E_WARNING, "Session cookies cannot be used when session.use_cookies is disabled");
+		RETURN_FALSE;
+	}
```

Comment:
Should we even bother with an exception in master? Wasn't that INI setting deprecated in 8.4?

Reply:
No, for BC reasons this INI setting was left unaffected: https://wiki.php.net/rfc/deprecate-get-post-sessions#sessionuse_cookies_off

Reply:
The rest of the code for settings checks seems to use warnings, so if we change it we should be consistent. Maybe for now let's not

Reply:
AAAAAAAAAA okay, then it makes sense, but maybe it should be a more cohesive change to convert ext/session warnings to exceptions.